app-editors/featherpad/featherpad-1.0.0.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit xdg cmake
+
+DESCRIPTION="Lightweight Qt5 Plain-Text Editor for Linux"
+HOMEPAGE="https://github.com/tsujan/FeatherPad"
+SRC_URI="https://github.com/tsujan/FeatherPad/archive/V${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="app-text/hunspell
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	dev-qt/qtnetwork:5
+	dev-qt/qtprintsupport:5
+	dev-qt/qtsvg:5
+	dev-qt/qtwidgets:5
+	dev-qt/qtx11extras:5
+	x11-libs/libX11"
+DEPEND="${RDEPEND}"
+BDEPEND="dev-qt/linguist-tools:5"
+
+S="${WORKDIR}/FeatherPad-${PV}"
```

Comment:
Please put the variable `S=` a line after `SRC_URI` (without extra line between them) as they are connected.

Reply:
Sure, I can do that. Is it something new? All other ebuilds under app-editors seems to do it similar to what I have used.

Reply:
You can always reference the `skel.ebuild` file located at the root of the tree, to know the basic order of variables.
The change itself is quite old (have been done [here](https://github.com/gentoo/gentoo/commit/0f936ec81f1dc20fa58ba40bc17989ee44810214)).

About the other ebuilds, I guess they just haven't been corrected since.

Reply:
Thank you for the `skel.ebuild` reference.